src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Config/ServiceBusOptions.cs

```diff
@@ -29,6 +29,13 @@ public ServiceBusOptions()
             };
 
             SessionHandlerOptions = new SessionHandlerOptions(ExceptionReceivedHandler);
+
+            // Set configuration for non-dynamic SKUs
+            string skuValue = Environment.GetEnvironmentVariable(Constants.AzureWebsiteSku);
+            if (!string.Equals(skuValue, Constants.DynamicSku, StringComparison.OrdinalIgnoreCase))
+            {
+                MessageHandlerOptions.MaxConcurrentCalls *= Environment.ProcessorCount;
+            }
```

Comment:
Are we doing this in response to an issue? Is this a recommendation from SB?

Reply:
Don't we have a tracking item for this that you can link to this bug? I think the work item across all triggers is to use more intelligent defaults across all our triggers, to improve performance.

What about SessionHandlerOptions? What is the default for PrefetchCount - should we also default that?

Reply:
tracking item is #27. `SessionHandlerOptions` doesn't have `PrefetchCount` but it have `MaxConcurrentSessions`. We don't want to change default `MaxConcurrentSessions` depending on core count.

Reply:
Ok